tests/arp/test_stress_arp.py

```diff
@@ -92,10 +119,16 @@ def test_ipv4_arp(duthost, garp_enabled, ip_and_intf_info, intfs_for_test,
                 logger.debug("Expected route number: {}, real route number {}"
                              .format(arp_avaliable, get_fdb_dynamic_mac_count(duthost)))
                 pytest_assert(wait_until(20, 1, 0,
-                                         lambda: abs(arp_avaliable - get_fdb_dynamic_mac_count(duthost)) < 100),
+                                         lambda: abs(arp_avaliable - get_fdb_dynamic_mac_count(duthost)) < 250),
```

Comment:
Also in your description, "For specific SKUs", have you tested on these skus and got the result? We observe that the total entries we add will no more than 10k, and we set the toleration percentage 1% here. Thus we get the number 100.

Reply:
Yes - with `100` the tests are extremely flaky, failing most of the time. At `200` it occasionally fails and at `250` it very rarely fails.
I think testing for a 99% load factor for a hash table is fairly unrealistic, especially when the MAC addresses in this test are increased linearly. The test is more about the hash function than the arp-handling logic at this point.